libbpf-tools/trace_helpers.c

```diff
@@ -500,3 +500,22 @@ bool kprobe_exists(const char *name)
 	fclose(f);
 	return false;
 }
+
+bool is_btf_enabled_for_kernel(void)
```

Comment:
maybe let's call them consistent with _exists pattern for fentry/fexit: vmlinux_btf_exists() and module_btf_exists()?

Reply:
I agree with you.
I changed the name of functions.
Thanks!